packages/core/content-manager/server/controllers/index.js

```diff
@@ -11,6 +12,7 @@ module.exports = {
   'collection-types': collectionTypes,
   components,
   'content-types': contentTypes,
+  init,
   relations,
   'single-types': singleTypes,
   uid,
```

Comment:
Do we need to look at deprecating some of these APIs because they are not calling in the admin panel anymore?

Reply:
Currently `/content-types` and `/components` are still being called in [this other hook](https://github.com/strapi/strapi/blob/f57ddb4cef9b815396828e0169a9cc4eecb45a11/packages/core/admin/admin/src/hooks/useModels/index.js#L5). But yes we could review each route individually to check if it is really needed. I would just prefer to do it in another PR if possible, as this one is getting big and I prefer incremental changes

Reply:
Not necessarily delete. You can use `JSdoc` so developers don't use it and they should use `init` instead etc. Then if it was a hook for example you could warn once like this https://github.com/strapi/design-system/blob/ab360f7a7d23dacb105f9b15984c4f7994a7e08e/packages/strapi-design-system/src/helpers/useIntersection.ts#L10 I haven't deleted the method, it just shows a warning to stop people using it.

Reply:
Deleting could be a breaking changes in some places (not necessarily here) so deprecating is a stepping stone to move people to new things without breaking a project 👍🏼